Packs/JamfProtect/Integrations/JamfProtectEventCollector/JamfProtectEventCollector.py

```diff
@@ -445,11 +445,11 @@ def get_events_alert_type(client: Client, start_date: str, max_fetch: int, last_
     client_event_type_func = client.get_alerts
     next_page = last_run.get("alert", {}).get("next_page", "")
 
-    demisto.debug(f"Jamf Protect- Fetching alerts from {created}")
+    demisto.debug(f"Jamf Protect- Fetching alerts from {created} with next page: {next_page}")
```

Comment:
Please remove all `Jamf Protect-` prefixes, they're not needed. DM for more info.